source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -858,11 +888,13 @@ JSON Schema. This operation creates a document similar to the following:
 Query for Documents on a Deterministically Encrypted Field
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Queries on encrypted fields can be done using traditional MongoDB driver
-methods as well. When a query is made using a client configured to use
-CSFLE, such as when a doctor in the MedcoMD management system searches
-for a patient with their SSN, the patient's data is returned
-unencrypted:
+.. image:: /figures/CSFLE_Read_Encrypted.png
+   :alt: Diagram that shows the data flow for reading field-level encrypted data
```

Comment:
Don't read operations need to use the JSON schema to know which data key to decrypt with?

![image](https://user-images.githubusercontent.com/15657698/70644474-14f76200-1c11-11ea-90e0-ac272c1d7c85.png)

Reply:
Yes, but the presence of the JSON Schema is represented by the top arrow "Validates query with automatic encryption configuration". We updated the doc to refer to the schema as "automatic encryption configuration (in JSON Schema format)" since it was a point of confusion in user research sessions.
Do you think it would be helpful to include a parenthetical (JSON Schema) under the current label? Otherwise, what would you suggest?

Reply:
If we updated the term in the guide then this should make sense. The JSON schema parenthetical might help but isn't necessary.